### ISSUE TITLE
Databasechangelog is not created with PostgreSQL two schemas

### ISSUE THREAD

**Issue by elghali** [liquibase/liquibase#1087]
**Version of Liquibase**
Apache License
Version 2.0, January 2004
**Liquibase Integration & Version**: <Pick one: CLI, maven, gradle, spring boot, servlet, etc.>
CLI
**Database Vendor & Version**:

PostgreSQL version 12.2
##Description
I'm trying to automate deployment to PostgreSQL using liquibase, basically I have a database "my_db" with two schemas "myschema1", "myschema2". I have two targets that I execute:

```install.bat install-myschema1-db``` with a url pointing to myschema1 schema

```install.bat install-myschema2-db``` with a url pointing to myschema2 schema

the first target executes correctly and creates all the needed tables, it also automatically creates the myschema1.databasechangelog table used by liquibase for logging.
However, when I try to execute the ```install-myschema2-db``` target it fails with ERROR:
> Error executing SQL SELECT MD5SUM FROM myschema2.databasechangelog WHERE MD5SUM IS NOT NULL LIMIT 1: ERROR: relation "myschema2.databasechangelog" does not exist

Basically, the installer scans for `databasechangelog` table and it find it in the database so it doesn't create, but when it tries to search in it , it tries to look in `myschema2.databasechangelog` but doesn't find it since it's created under `myschema1`

##Steps to Reproduce
`"%ANT_HOME%\bin\ant.bat" -f install.xml -Denv.file=install.properties %CMD_LINE_ARGS%`

%CMD_LINE_ARGS% is the target to execute which in this case is ```install-myschema2-db```

**Comment by molivasdat** [comment]
Hi @elghali .  Thanks for submitting this issue.  Can you supply a little more details on recreation steps you used to find this issue.
I have update the comment section with a few more fields. If you could update the fields with additional info that would help us greatly.  I have tagged them with your github id in the comments field for you to replace your comments.

**Comment by FBurguer** [comment]
Liquibase lets you create a different `databasechaneglog` table on different schemas, so it doesnt look like a liquibase error (at least on `4.14`). Try upgrading your liquibase version, if you still have any issue, you can try re-opening the issue. Thanks!
